tasks/config/karma.js

```diff
@@ -1,5 +1,27 @@
+import { times, zipObject } from 'lodash';
+
+function ciShardConfigs(totalShards) {
+  return zipObject(times(totalShards, i => {
```

Comment:
`zipObject` and `times` are a little more confusing for loop and assignment to an object.